src/f4enix/input/d1suned.py

```
@@ -38,7 +41,7 @@ def __init__(
         nsc: int,
         irr_schedules: list[Irradiation],
         header: str | None = None,
-        formatting: list[int] = [11, 14, 13, 9],
+        formatting: list[int] = [14, 14, 13, 11],
```

Ruff (0.14.8)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)